pom.xml

```diff
@@ -18,6 +18,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
+        <juipter.version>5.10.2</juipter.version>
```

Comment:
There is a typo here and everywhere the variable is used. It should be `jupiter` instead of `juipter`.

Reply:
updated